apps/core/components/register-customer-form/_actions/login.ts

```diff
@@ -0,0 +1,28 @@
+'use server';
+
+import { isRedirectError } from 'next/dist/client/components/redirect';
+
+import { signIn } from '~/auth';
+
+export const login = async (
+  email: FormDataEntryValue | null,
+  password: FormDataEntryValue | null,
+) => {
+  try {
+    const singin = await signIn('credentials', {
+      email,
+      password,
+      redirectTo: '/account',
```

Comment:
I believe we already redirect to account but I might be wrong.

Reply:
Yes, we have, but `submitLoginForm` action used for form and has different arguments. And using it here requires huge refactoring and can add mess in currently existed action.

Reply:
This is okay then!